librz/analysis/var.c

```diff
@@ -47,10 +47,17 @@ RZ_API bool rz_analysis_function_rebase_vars(RzAnalysis *a, RzAnalysisFunction *
 // remove all other vars that are overlapped by var and are at the offset of one of its struct members
 static void shadow_var_struct_members(RzAnalysisVar *var) {
 	RzBaseType *btype = rz_type_db_get_base_type(var->fcn->analysis->typedb, var->type);
-	if (!btype || btype->kind != RZ_BASE_TYPE_KIND_STRUCT) {
```

Comment:
Please avoid changes in RzTypes-related area since I work on it in my PR. This particular one is fine, just don't do more, please.

Reply:
i was just fixing memory leaks.